### ISSUE TITLE
Issue when installing the apps

### ISSUE THREAD

**Issue by LawlietKM** [Drovosek01/adobe-packager#57]
When I try to install a downloaded app it says the following error:

Error: SyntaxError: JSON Parse error: Unexpected EOF (-2700)

**Comment by omppumetro** [comment]
Same issue here with installer apps created with this tool.
Seems like that Creative Cloud version 5.10.0.573 broke this functionality.
I can install and update via CC app normally.

**Comment by omppumetro** [comment]
Quick peek into those application.json files did not show anything broken, maybe CC requires them installers to be signed?

**Comment by omppumetro** [comment]
This is what the Adobe Setup writes to log after installation fails:

```
03/03/23 20:53:21:054 | [FATAL] | 73070 | HDSetup | HDSetup | AppContext |  | HDSetup | 13568918 | 0 | Adobe Setup is not Authorized
03/03/23 20:53:21:054 | [INFO] | 73070 | HDSetup | HDSetup | AppContext |  | HDSetup | 13568918 | 0 | Exit Code: -1

```
I guess that Adobe Setup needs valid installer components and this adobe-packager script cannot provide those.
Not sure how this could be fixed as getting validation from Adobe for those components may be tricky, even impossible.
It was useful tool as long it worked, we are back to using original installers from Adobe.

**Comment by Damnbebo** [comment]
Hey I had the same issue and I was able to solve it, I would leave the steps I follow to fix

Step 1. Install Creative Cloud 5.7.0.1307 or install it from here (https://mega.nz/file/yaZBhDoB#p3nTh7-Bdg3Li1SUaAjcYd33Zh6GCkLJ87LJLtDaW9Y) This link is mine I uploaded the one I installed so do not worry .

Step 2. Before you start, please click uninstall on the installation screen for the old version of Creative.

Step 3. After the installation is done go to /Library/Application Support/Adobe and locate the folder name "Caps" and delete it or send it to Trash. "You can easily go to the location by Holding Shift+Command+G on your desktop."

Step 4. just try to install the Adobe Application you were trying to install before and it should work 100% with no issue.

I just did this and worked 100% for me.
if you have any questions add me on Discord 💢FW ᵇᵉᵇᵒ💊#0880

**Comment by vermar-js** [comment]
> Hey I had the same issue and I was able to solve it, I would leave the steps I follow to fix
> 
> Step 1. Install Creative Cloud 5.7.0.1307 or install it from here (https://mega.nz/file/yaZBhDoB#p3nTh7-Bdg3Li1SUaAjcYd33Zh6GCkLJ87LJLtDaW9Y) This link is mine I uploaded the one I installed so do not worry .
> 
> Step 2. Before you start, please click uninstall on the installation screen for the old version of Creative.
> 
> Step 3. After the installation is done go to /Library/Application Support/Adobe and locate the folder name "Caps" and delete it or send it to Trash. "You can easily go to the location by Holding Shift+Command+G on your desktop."
> 
> Step 4. just try to install the Adobe Application you were trying to install before and it should work 100% with no issue.
> 
> I just did this and worked 100% for me. if you have any questions add me on Discord 💢FW ᵇᵉᵇᵒ💊#0880

It worked for me with my 2020 Adobe program and Adobe Zii 5.6.1. I had this error with JSON and now it's installed

**Comment by kevin25** [comment]
> Hey I had the same issue and I was able to solve it, I would leave the steps I follow to fix
> 
> Step 1. Install Creative Cloud 5.7.0.1307 or install it from here (https://mega.nz/file/yaZBhDoB#p3nTh7-Bdg3Li1SUaAjcYd33Zh6GCkLJ87LJLtDaW9Y) This link is mine I uploaded the one I installed so do not worry .
> 
> Step 2. Before you start, please click uninstall on the installation screen for the old version of Creative.
> 
> Step 3. After the installation is done go to /Library/Application Support/Adobe and locate the folder name "Caps" and delete it or send it to Trash. "You can easily go to the location by Holding Shift+Command+G on your desktop."
> 
> Step 4. just try to install the Adobe Application you were trying to install before and it should work 100% with no issue.
> 
> I just did this and worked 100% for me. if you have any questions add me on Discord 💢FW ᵇᵉᵇᵒ💊#0880

It can be downloaded here https://ccmdl.adobe.com/AdobeProducts/KCCC/CCD/5_10_0/osx10/ACCCx5_7_0_1307.dmg

**Comment by mackonsti** [comment]
Thanks @kevin25 but the URL is not valid :(

**Comment by dimasxp** [comment]
> Thanks @kevin25 but the URL is not valid... did you reconstruct it yourself? Folder is 5_10_0 but version is 5_7_0...
> Found these ones that work:
> https://ccmdl.adobe.com/AdobeProducts/KCCC/CCD/5_7_1/osx10/ACCCx5_7_1_1.dmg
> https://ccmdl.adobe.com/AdobeProducts/KCCC/CCD/5_7_0/osx10/ACCCx5_7_0_1307.dmg

receive this error on m1 ,  did you manage to install?
![CleanShot 2023-04-11 at 20 13 05@2x](https://user-images.githubusercontent.com/7904186/231173961-81cd726d-7b89-47fa-be62-f30a96cc2635.jpg)

**Comment by LunaAmt** [comment]
> > Hey I had the same issue and I was able to solve it, I would leave the steps I follow to fix
> > Step 1. Install Creative Cloud 5.7.0.1307 or install it from here (https://mega.nz/file/yaZBhDoB#p3nTh7-Bdg3Li1SUaAjcYd33Zh6GCkLJ87LJLtDaW9Y) This link is mine I uploaded the one I installed so do not worry .
> > Step 2. Before you start, please click uninstall on the installation screen for the old version of Creative.
> > Step 3. After the installation is done go to /Library/Application Support/Adobe and locate the folder name "Caps" and delete it or send it to Trash. "You can easily go to the location by Holding Shift+Command+G on your desktop."
> > Step 4. just try to install the Adobe Application you were trying to install before and it should work 100% with no issue.
> > I just did this and worked 100% for me. if you have any questions add me on Discord 💢FW ᵇᵉᵇᵒ💊#0880
> 
> It can be downloaded here https://ccmdl.adobe.com/AdobeProducts/KCCC/CCD/5_10_0/osx10/ACCCx5_7_0_1307.dmg

thanks, man, you are a life saver, you the best

**Comment by MarcosRic** [comment]
URL para Apple ARM Silicon: https://ccmdl.adobe.com/AdobeProducts/KCCC/CCD/5_7_0/macarm64/ACCCx5_7_0_1307.dmg

**Comment by kevin25** [comment]
You can download it here either https://ccmdl.adobe.com/AdobeProducts/KCCC/CCD/5_7_0/osx10/ACCCx5_7_0_1307.dmg

**Comment by elmimmo** [comment]
As detailed by Adobe at their document [Issue installing package created before February 2023](https://helpx.adobe.com/enterprise/kb/rum-release-notes/issue-installing-package-created-before-feb-2023.html)

**Comment by mischanja708** [comment]
The check can be disabled in any new version of Creative Cloud by binary patching "/Library/Application Support/Adobe/Adobe Desktop Common/HDBox/Setup".

**Comment by Antibioticss** [comment]
Yes, I already know how to patch it, but modifying that file may cause other problem. 
Maybe we can make a backup and revert it after installation.
